docs/about.rst

```diff
@@ -58,13 +58,17 @@ We think that's a pretty fair deal, and we're big believers in open source. If y
 Presentations
 -------------
 
-Yellowbrick has enjoyed the spotlight at a few conferences and in several presentations. We hope that these videos, talks, and slides will help you understand Yellowbrick a bit better.
+Yellowbrick has enjoyed the spotlight in several presentations at recent conferences. We hope that these notebooks, talks, and slides will help you understand Yellowbrick a bit better.
 
-Videos:
+Conference Presentations (videos):
     - `Visual Diagnostics for More Informed Machine Learning: Within and Beyond Scikit-Learn (PyCon 2016) <https://youtu.be/c5DaaGZWQqY>`_
     - `Yellowbrick: Steering Machine Learning with Visual Transformers (PyData London 2017) <https://youtu.be/2ZKng7pCB5k>`_
 
+Jupyter Notebooks:
```

Comment:
Add a new "Jupyter Notebooks" section and a link to Ian Ozsvald's ML regression tutorial. I think we can come up with a better title for this section but it might become clearer as we start to add new resources and get a better idea of how we'd like to structure everything.